test/demo/eventListeners.inttest.ts

```diff
@@ -0,0 +1,170 @@
+import 'jest';
+import {Application, Window as FinWindow} from 'hadouken-js-adapter';
+
+import {NotificationClickedEvent, Notification, NotificationOptions, NotificationButtonClickedEvent, NotificationClosedEvent} from '../../src/client';
+
+import {fin} from './utils/fin';
+import * as notifsRemote from './utils/notificationsRemoteExecution';
+import {getCardsByNotification, isCenterShowing} from './utils/notificationCenterUtils';
+import {delay} from './utils/delay';
+
+const defaultNoteOptions: NotificationOptions = {
+    body: 'Test Notification Body',
+    title: 'Test Notification Title',
+    buttons: [
+        {title: 'Button 1'}
+    ]
+};
+
+describe('Click listeners', () => {
+    beforeAll(async () => {
+        // Make sure the center is showing
+        if (!(await isCenterShowing())) {
+            await notifsRemote.toggleNotificationCenter({uuid: 'test-app', name: 'test-app'});
```

Comment:
Can we create a constant for this identity? These tests seem to have a convention of using `testApp` for windows that are created for the lifespan of the test - having the persistent window also be called `test-app` might be confusing.

Reply:
This is in-line with what we do in FDC3: manager is just called 'test-app', and all other are some variation of `[service-name]-test-app-[count]`